### DISCUSSION TITLE
Remove files from LFS

### DISCUSSION THREAD

**Discussion by DigitalPath2000** [community/community#137280] [Actions]
### Select Topic Area

General

### Body

Hi,

I'm trying to delete files from the LFS (surprisingly difficult!!). I went ahead to the repo and changed the gitattributes file, commenting out all the lines that instructed the repo to deposit files in the LFS. It doesn't seem to have solved my problem, and my LFS is still full of files from that repo.

I'm at my wits end. Any advice on how I could remove the files from the LFS would be a big help.

**Comment by davevad93** [comment]
Hi @DigitalPath2000 ,

Check the docs about [removing files from Git Large File Storage](https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage), there you can see the steps to do it.
